Firestore/Source/Model/FSTFieldValue.mm

```diff
@@ -1037,7 +985,9 @@ - (id)value {
     case FieldValue::Type::Double:
     case FieldValue::Type::Timestamp:
     case FieldValue::Type::ServerTimestamp:
+      HARD_FAIL("TODO(rsgowman): implement");
     case FieldValue::Type::String:
+      return util::WrapNSStringNoCopy(self.internalValue.string_value());
```

Comment:
Nit: consider omitting the `NoCopy` part, tying the lifetime of the return value to the lifetime of the field value makes me a little wary.